connman/src/service.c

```diff
@@ -3433,6 +3433,27 @@ static void append_properties(DBusMessageIter *dict, dbus_bool_t limited,
 				GET_DEFAULT_ACCESS_ACCESS);
 }
 
+static void append_properties_updated(DBusMessageIter *dict,
+	struct connman_service *service)
+{
+	switch (service->type) {
+	case CONNMAN_SERVICE_TYPE_UNKNOWN:
+	case CONNMAN_SERVICE_TYPE_SYSTEM:
+	case CONNMAN_SERVICE_TYPE_GPS:
+	case CONNMAN_SERVICE_TYPE_VPN:
+	case CONNMAN_SERVICE_TYPE_P2P:
+	case CONNMAN_SERVICE_TYPE_CELLULAR:
+	case CONNMAN_SERVICE_TYPE_ETHERNET:
+	case CONNMAN_SERVICE_TYPE_BLUETOOTH:
+	case CONNMAN_SERVICE_TYPE_GADGET:
+		break;
+	case CONNMAN_SERVICE_TYPE_WIFI:
+		if (service->network != NULL)
```

Comment:
For simplicity this check can be just `if (service->network)`

Reply:
Which is also suggested by [coding style M13](https://github.com/sailfishos/connman/blob/master/connman/doc/coding-style.txt#L247).

Reply:
Nice catch, thanks, I have fixed that